src/main/java/edu/iastate/ece/sd/sdmay2126/application/Application.java

```diff
@@ -0,0 +1,11 @@
+package edu.iastate.ece.sd.sdmay2126.application;
+
+/**
+ * Specified a KBase application/simulation.
+ */
+public enum Application {
```

Comment:
I think `ApplicationType` makes more sense as a name to me. `Application` makes it seem like an object that will run some logic.